app/services/api/webflow-helpers.ts

```diff
@@ -191,7 +209,12 @@ export const convertScheduleToScheduleCard = (
   day: string,
 ): ScheduleCardProps[] => {
   const daySchedule: ScheduledEvent[] = groupBy("day")(scheduleData ?? [])?.[day] ?? []
-  return daySchedule.sort(sortByTime).map(convertScheduleToCardProps)
+  const sortedDaySchedule = daySchedule
+    .sort(sortByTime)
+    .map(convertScheduleToCardProps)
+    .filter(Boolean)
+  console.tron.log({ day, daySchedule: daySchedule.sort(sortByTime), sortedDaySchedule })
```

Comment:
We will probably want to remove this before merging, or hide behind __DEV__ flag

Reply:
I did remove this console.tron call in my other branch: https://github.com/infinitered/ChainReactApp2023/pull/125

Reply:
its was left over from debugging, removing in next push